givitsite/friendreq/views.py

```diff
@@ -24,13 +22,16 @@ def itemRequest_create_view(request):
         return render(request, 'itemRequestform.html',context)
 
 
-def requestItem(request):
+def requestItem(request):      
     # on match
     if request.method == 'POST':
-        id = request.POST["id"]
-        matchItem = ItemsFound.objects.filter(pk=id).update(match=True)
```

Comment:
if all the changes below are related you may want to group them together as a transaction so that one will not happen without the other

Reply:
do you mean to create a kind of dependence between the update operations?

Reply:
I mean to use a transaction - Its a way to group database operations together so they are done in an atomic way. Its one of the more important DB concepts and the ORM certainly provides you with a way to do that. Probably more then one way.

Reply:
fixed